docs/product/components/labels.html

```diff
@@ -14,15 +14,15 @@
     {% header "h2", "Base style" %}
     <div class="stacks-preview">
 {% highlight html %}
-<form class="d-flex gs4 gsy fd-column">
+<form class="d-flex gx4 fd-column">
```

Comment:
## Change requested

Since `<label>` elements are rendered as `display: table`, the gap property doesn't apply. Any `<label>` with parents that include a gap property need either to have a margin added or have their `display` property altered to honor `gap`.

Reply:
Also here cannot see visual differences.

Reply:
Yep, not sure what issue I saw here yesterday! Sorry for the noise!